r/adbcdrivermanager/src/driver_base.h

```diff
@@ -266,7 +266,9 @@ class ObjectBase {
 
   AdbcStatusCode CSetOptionBytes(const char* key, const uint8_t* value, size_t length,
                                  AdbcError* error) {
-    std::basic_string<uint8_t> cppvalue(value, length);
+    std::vector<uint8_t> cppvalue(length);
+    memcpy(cppvalue.data(), value, length);
```

Comment:
FWIW, I think it should work to write

```cpp
std::vector<uint8_t> cppvalue(value, value + length);
```

(constructor (5) here https://en.cppreference.com/w/cpp/container/vector/vector)